scss/private/breakpoints/_breakpoints.scss

```diff
@@ -87,6 +87,10 @@
     @include mappy-bp(map-get($breakpoints, $breakpoint)) {
       @content;
     }
+  } @else if $lib == 'mq' {
```

Comment:
It would be great if `mq` is `sass-mq`. This lets other people switch libraries knowing that they're gonna work with Sass MQ.

Reply:
Hi, thanks for reviewing. The reason I used mq that's the name of the mixin in the sass mq library and in typi it checks for if the mixin exists by that string (in utils.scss). Am I misunderstanding or is that not how it works.

Reply:
Oh cool! I didn't know SassMQ uses `mq'. This works then.